### PR TITLE
Update to graph-cli / graph-ts 0.13.x

### PR DESCRIPTION
**Pull request by Jannis**
* Updates dependencies to graph-cli 0.13.2 and graph-ts 0.13.0
* Introduces `indexed` hints for indexed parameters in event signatures
* Bumps the mapping `apiVersion` from 0.0.2 to 0.0.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
   "name": "gitcoin-subgraph",
   "version": "0.0.1",
-  "repository": "https://github.com/santteegt/gitcoin-subgraph",
+  "repository": "https://github.com/jannis/gitcoin-subgraph",
```
Comment (santteegt) [inline package.json]: Please remove the references to your repo

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   "license": "MIT",
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ santteegt/gitcoin",
-    "create-local": "graph create --node http://localhost:8020/ santteegt/gitcoin",
-    "remove-local": "graph remove --node http://localhost:8020/ santteegt/gitcoin",
-    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 santteegt/gitcoin"
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ jannis/gitcoin",
```
Comment (santteegt) [inline package.json]: Please remove references to your repo in scripts

**Review by santteegt — CHANGES_REQUESTED** [review]
Hi @Jannis,

Thanks for your PR. Please check my comments on the `package.json`

**Comment by Jannis** [comment]
@santteegt Oops, that was an oversight on my part. Fixed!

**Pull request merged**